### PR TITLE
Feature/slp view

### LINKED ISSUE #30: Semester long project view
Make the view for Semester Long Project

### PR DESCRIPTION
**Pull request by bsalgado98**
Closes #30

### CODE REVIEW

**Comment by carpenterd777** [comment]
Screen Shot 2020-02-14 at 9 16 59 AM
On mobile the text in each of the tile headers gets squished together and becomes very difficult to read. Not sure if mobile design was planned for a future PR though.

I think you might have forgotten to write `open-together-tube.md`. When I click on the openTogetherTube tile, the whole site crashes

**Comment by dyc3** [comment]
Please rebase this branch on master to resolve the merge conflicts

**Comment by dyc3** [comment]
I don't think the current state of this branch gives a good user experience. I'm working on another branch to try to do a scrolling based router, where you can scroll on the page and it changes the route based on where on the page you scroll.

**Comment by dyc3** [comment]
I don't think the current state of this branch gives a good user experience on desktop, mainly because of the behavior of the `ProjectWall` component. I'm working on another branch to try to do a scrolling based router, where you can scroll on the page and it changes the route based on where on the page you scroll.

File: sec.club/src/views/SLP/Projects.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    year: "2019",
+    markdown: "slp/open-together-tube.md"
+  },
+  {
+    path: "stevens-web-extension",
+    img: secLogo,
+    name: "Stevens Web Extension",
+    semester: "Spring",
+    year: "2020",
+    markdown: "slp/quick-quack.md"
```
Comment (carpenterd777) [inline sec.club/src/views/SLP/Projects.js]: Why does the stevens web extension reference the quick-quack markdown file?

Comment (dyc3) [reply sec.club/src/views/SLP/Projects.js]: Didn't bother to change it. We don't have text content for it yet anyway

Comment (carpenterd777) [reply sec.club/src/views/SLP/Projects.js]: Fair enough

**Pull request merged**